mermaid/flowchart/examples_test.go

```
@@ -0,0 +1,53 @@
+	).
+		NodeWithText("A", "Node A").
+		StadiumNode("B", "Node B").
+		SubroutineNode("C", "Node C").
+		DatabaseNode("D", "Database").
+		LinkWithArrowHead("A", "B").
+		LinkWithArrowHeadAndText("B", "D", "send original data").
+		LinkWithArrowHead("B", "C").
+		DottedLinkWithText("C", "D", "send filtered data").
+		String()
+
+	markdown.NewMarkdown(os.Stdout).
```

golangci-lint (1.62.2)

[low] G104: Errors unhandled.

(gosec)